From c082a89c5b08a4755f8ec9ec4d7f8b2db97b11d4 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:01:21 -0400
Subject: [usb] host: delete chipidea dependency

Message-id: <1410887463-20674-10-git-send-email-dzickus@redhat.com>
Patchwork-id: 95560
O-Subject: [RHEL7 PATCH 009/591] usb: host: delete chipidea dependency
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit a746c286f32126ce6f0f98c111bb7ea15943e083
Author: Peter Chen <peter.chen@freescale.com>
Date:   Tue Sep 17 12:37:18 2013 +0800

    usb: host: delete chipidea dependency

    Now, chipidea host has already depended on USB_EHCI_HCD

    Acked-by: Alan Stern <stern@rowland.harvard.edu>
    Signed-off-by: Peter Chen <peter.chen@freescale.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/host/Kconfig b/drivers/usb/host/Kconfig
index 40e3d34..10fd455 100644
--- a/drivers/usb/host/Kconfig
+++ b/drivers/usb/host/Kconfig
@@ -54,7 +54,7 @@ config USB_EHCI_HCD
 
 config USB_EHCI_ROOT_HUB_TT
  bool "Root Hub Transaction Translators"
- depends on USB_EHCI_HCD || USB_CHIPIDEA_HOST
+ depends on USB_EHCI_HCD
  ---help---
    Some EHCI chips have vendor-specific extensions to integrate
    transaction translators, so that no OHCI or UHCI companion
@@ -66,7 +66,7 @@ config USB_EHCI_ROOT_HUB_TT
 
 config USB_EHCI_TT_NEWSCHED
  bool "Improved Transaction Translator scheduling"
- depends on USB_EHCI_HCD || USB_CHIPIDEA_HOST
+ depends on USB_EHCI_HCD
  default y
  ---help---
    This changes the periodic scheduling code to fill more of the low
-- 
1.7.1